pages/design-studio.md

```diff
@@ -38,5 +38,5 @@ No PRA implications. If conducted with nine or fewer members of the public, the
 
 ## Additional resources
 
-A [presentation by Todd Zaki Warfel.](https://vimeo.com/37861987/)
+A [presentation by Todd Zaki Warfel](https://vimeo.com/37861987/)
  explaining what Design Studio is. Todd Zaki Warfel.
```

Comment:
AH! I see it now, sorry about that. Last tiny annoying question, but while we're splitting hairs...should this be `explaining what *a* design studio is` ?